backend/routing/users.js

```diff
@@ -43,6 +43,43 @@ const usersController = require('../controllers/users.js')(DAL);
     }
   });
 
+  /**
+   * @api {post} /api/register Request For Register User
+   * @apiName RegisterUser
+   * @apiGroup Users
+   *
+   * @apiSuccess {Object[]} users           List of user register data.
```

Comment:
This is response fields. Do this request return array of objects ( {Object[]} )?
See examples to find out how to add request parameters http://apidocjs.com/#examples